src/filter/http_context.rs

```diff
@@ -195,6 +149,23 @@ impl Filter {
         Some(res)
     }
 
+    fn get_typed_property(&self, path: &str) -> TypedProperty {
```

Comment:
This should then return an `Option<TypedProperty>` and act accordingly on the call site, wdyt?